addons/default/visiosoft/advs-module/resources/views/list/gallery.twig

```diff
@@ -32,7 +32,8 @@
                             <p class="cat-data mb-1">{{ adv.cat1_name }} /{{ adv.cat2_name }}</p>
                         {% endif %}
                         {% if showPrice %}
-                            <p class="price-data mb-1">
+                            <p class="price-data mb-1
```

Comment:
we need to make function for it @muammertop

Reply:
do we have currency_format

Reply:
Should it be checked in currency_format?

Reply:
We dont have currency format @fatihalp